frontend/i18n-scripts/i18n-to-po.js

```diff
@@ -84,9 +84,9 @@ function processFile(fileName, packageDir, language) {
             save(
               path.join(
                 __dirname,
-                `./../po-files/${language}/${path.basename(fileName)}_package=${
-                  packageDir.split('/')[0]
-                }.po`,
+                `./../po-files/${language}/${packageDir.split('/')[0]}_${path.basename(
```

Comment:
I would make this two underscores `__` like BEM. Then we don't accidentally do the wrong thing if a package uses an underscore in its namespace. It makes it more obvious it's a meaningful separator, too.